# HG changeset patch
# User andrew
# Date 1598596701 -3600
#      Fri Aug 28 07:38:21 2020 +0100
# Node ID 7323b2187a50e3d04cdc3b0e35bac32875e34e09
# Parent  b73ff161ed0a57a46fdb3ed51291e39c67d880f5
8251120, PR3793: [8u] HotSpot build assumes ENABLE_JFR is set to either true or false
Summary: Only test for ENABLE_JFR being true, and assume undefined == false
Reviewed-by: neugens

diff --git openjdk.orig/hotspot/make/aix/makefiles/buildtree.make openjdk/hotspot/make/aix/makefiles/buildtree.make
--- openjdk.orig/hotspot/make/aix/makefiles/buildtree.make
+++ openjdk/hotspot/make/aix/makefiles/buildtree.make
@@ -111,7 +111,7 @@
 endif
 endif
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 ALWAYS_EXCLUDE_DIRS += -o -name jfr
 endif
 
diff --git openjdk.orig/hotspot/make/aix/makefiles/vm.make openjdk/hotspot/make/aix/makefiles/vm.make
--- openjdk.orig/hotspot/make/aix/makefiles/vm.make
+++ openjdk/hotspot/make/aix/makefiles/vm.make
@@ -147,7 +147,7 @@
 LIBJVM_DEBUGINFO   = lib$(JVM).debuginfo
 LIBJVM_DIZ         = lib$(JVM).diz
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 EXCLUDE_JFR_PATHS:= -o -name jfr -prune
 endif
 
diff --git openjdk.orig/hotspot/make/bsd/makefiles/buildtree.make openjdk/hotspot/make/bsd/makefiles/buildtree.make
--- openjdk.orig/hotspot/make/bsd/makefiles/buildtree.make
+++ openjdk/hotspot/make/bsd/makefiles/buildtree.make
@@ -113,7 +113,7 @@
 endif
 endif
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 ALWAYS_EXCLUDE_DIRS += -o -name jfr
 endif
 
diff --git openjdk.orig/hotspot/make/bsd/makefiles/vm.make openjdk/hotspot/make/bsd/makefiles/vm.make
--- openjdk.orig/hotspot/make/bsd/makefiles/vm.make
+++ openjdk/hotspot/make/bsd/makefiles/vm.make
@@ -165,7 +165,7 @@
   LIBJVM_DIZ         = lib$(JVM).diz
 endif
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 EXCLUDE_JFR_PATHS:= -o -name jfr -prune
 endif
 SPECIAL_PATHS:=adlc c1 gc_implementation opto shark libadt
diff --git openjdk.orig/hotspot/make/linux/makefiles/buildtree.make openjdk/hotspot/make/linux/makefiles/buildtree.make
--- openjdk.orig/hotspot/make/linux/makefiles/buildtree.make
+++ openjdk/hotspot/make/linux/makefiles/buildtree.make
@@ -112,7 +112,7 @@
 endif
 endif
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 ALWAYS_EXCLUDE_DIRS += -o -name jfr
 endif
 
diff --git openjdk.orig/hotspot/make/linux/makefiles/vm.make openjdk/hotspot/make/linux/makefiles/vm.make
--- openjdk.orig/hotspot/make/linux/makefiles/vm.make
+++ openjdk/hotspot/make/linux/makefiles/vm.make
@@ -158,7 +158,7 @@
 LIBJVM_DEBUGINFO   = lib$(JVM).debuginfo
 LIBJVM_DIZ         = lib$(JVM).diz
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 EXCLUDE_JFR_PATHS:= -o -name jfr -prune
 endif
 SPECIAL_PATHS:=adlc c1 gc_implementation opto shark libadt
diff --git openjdk.orig/hotspot/make/solaris/makefiles/buildtree.make openjdk/hotspot/make/solaris/makefiles/buildtree.make
--- openjdk.orig/hotspot/make/solaris/makefiles/buildtree.make
+++ openjdk/hotspot/make/solaris/makefiles/buildtree.make
@@ -103,7 +103,7 @@
 endif
 endif
 
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 ALWAYS_EXCLUDE_DIRS += -o -name jfr
 endif
 
diff --git openjdk.orig/hotspot/make/solaris/makefiles/vm.make openjdk/hotspot/make/solaris/makefiles/vm.make
--- openjdk.orig/hotspot/make/solaris/makefiles/vm.make
+++ openjdk/hotspot/make/solaris/makefiles/vm.make
@@ -172,7 +172,7 @@
 
 LIBJVM_DEBUGINFO   = lib$(JVM).debuginfo
 LIBJVM_DIZ         = lib$(JVM).diz
-ifeq ($(ENABLE_JFR),false)
+ifneq ($(ENABLE_JFR),true)
 EXCLUDE_JFR_PATHS:= -o -name jfr -prune
 endif
 SPECIAL_PATHS:=adlc c1 dist gc_implementation opto shark libadt
